### ISSUE TITLE
Add support for IE11

### ISSUE THREAD

**Issue by xet7** [wekan/wekan#1031]
@nztqa

At https://github.com/wekan/wekan/issues/784#issuecomment-299787407 you commented:

> Add support for IE11
> IE11 keeps "Trident" in it's UA string, but drops MSIE.
```
if(userAgent.indexOf('msie') >= 0 || userAgent.indexOf('trident') >= 0 || userAgent.indexOf('chrome') >= 0)
```

Do you mean generally in Wekan? Or is it somehow related to #784 ?

**Comment by nztqa** [comment]
this commitment https://github.com/wekan/cfs-access-point/commit/8c02e16394472537aef65ef216bae40b55dfff10#diff-324b24b854cc477ca6a307b0117ea76cR428

**Comment by xet7** [comment]
Fixed in ada9478ed7cb5967c6270d3c877cb8dc46fef4d2
